options/locale/locale_en-US.ini

```diff
@@ -3258,6 +3258,14 @@ container.layers = Image Layers
 container.labels = Labels
 container.labels.key = Key
 container.labels.value = Value
+debian.registry = Setup this registry from the command line:
+debian.registry.info = Choose <code>&lt;distribution&gt;</code> and <code>&lt;component&gt;</code> from the list below.
```

Comment:
Can we maybe not add HTML hardcoded into new translations and add it in as parameter instead?

Reply:
Agree, we should not allow HTML in new translations. Use the `printf` method as was done in https://github.com/go-gitea/gitea/pull/24397 to render HTML into the final string.

Reply:
The `printf` does only work if the content of the HTML is not translated.
```
{{$.locale.Tr "dummy" (printf "<a href='%[1]s'>%[2]s</a>" ($some_link|Escape) (put another $.locale.Tr here?)) | Safe}}
```
I don't think we should do nested translation.

Reply:
We can use other syntax like "$distribution" to avoid the `&lt;` in translation string.

Many translators can't handle it correctly.

And even worse, the `;` is comment for INI, crowdin will only see `debian.registry.info = Choose <code>&lt` and generates output like ``debian.registry.info = Choose <code>&lt ; ....` (yes, there might be a space)

Reply:
Hmm yes that's a bit unwieldly but should work. Could be cleaned up with use of in-template `$variables`. Eventually we should remove all HTML from translation.